packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -119,7 +118,7 @@ export default class RelationControl extends React.Component {
       const metadata = {};
       const searchFieldsArray = getSearchFieldArray(field.get('search_fields'));
       const { payload } = await query(forID, collection, searchFieldsArray, '', file);
```

Comment:
I would say that this is not idiomatic usage of redux action. One shouldn't rely on the return value of the action and get all the data only from props.

Reply:
Completely agree 💯